src/clr-angular/data/datagrid/datagrid.module.ts

```diff
@@ -47,6 +47,10 @@ import { DatagridRowRenderer } from './render/row-renderer';
 import { WrappedCell } from './wrapped-cell';
 import { WrappedColumn } from './wrapped-column';
 import { WrappedRow } from './wrapped-row';
+import { ClrDatagridDetail, ClrDatagridDetailDirective } from './datagrid-detail';
```

Comment:
Not a nit, just a question: is ther a reason to keep component and directive together in the same file? Its the first time I've seen us organize like that.

Reply:
I've changed this already, haven't pushed it.